samples/camera-capture/app.js

```diff
@@ -23,10 +23,43 @@ Author: Eric Bidelman (ericbidelman@chromium.org)
  *
  * @see http://developer.chrome.com/apps/manifest.html#permissions
  */
+
```

Comment:
Do you mind updating comment below mentioning we request both video and audio since this is not the case anymore?